alf/examples/actor_critic.py

```diff
@@ -1,229 +0,0 @@
-# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-r"""Train using ActorCriticAlgorithm.
-
-To run actor_critic on gym CartPole:
-```bash
-python actor_critic.py \
-  --root_dir=~/tmp/cart_pole \
-  --alsologtostderr \
-  --gin_file=ac_cart_pole.gin
-```
-
-You can visualize playing of the trained model by running:
-```bash
-python actor_critic.py \
-  --root_dir=~/tmp/cart_pole \
-  --alsologtostderr \
-  --gin_file=ac_cart_pole.gin \
-  --play
-```
-
-To run on SocialBot CartPole:
-```bash
-python actor_critic.py \
-  --root_dir=~/tmp/socialbot-cartpole \
-  --alsologtostderr \
-  --gin_param='create_environment.env_name="SocialBot-CartPole-v0"' \
-  --gin_param='create_environment.num_parallel_environments=16' \
-  --gin_param='create_environment.env_load_fn=@suite_socialbot.load' \
-  --gin_file=ac_cart_pole.gin
-```
-
-To Run on SocailBot SimpleNavigation
```

Comment:
Maybe add these removed examples of running command to main.py? Sometimes they are useful.

Reply:
All examples run in the same way
`python main.py --root_dir=${root_dir} --gin_file=${gin_flie} [--gin_param=...]`